libraries/core_libs/network/include/network/tarcap/packets_handlers/common/ext_syncing_packet_handler.hpp

```diff
@@ -45,9 +45,17 @@ class ExtSyncingPacketHandler : public PacketHandler {
   std::pair<bool, std::unordered_set<blk_hash_t>> checkDagBlockValidation(const DagBlock &block) const;
   std::shared_ptr<TaraxaPeer> getMaxChainPeer();
 
+ private:
+  void requestPendingDagBlocks(const dev::p2p::NodeID &node_id);
+
```

Comment:
this seems to be unused

Reply:
there is a lot of problems with the automatic merge in the networking code, I am working on manually fixing these